Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -380,24 +364,6 @@ open class CallViewModel: ObservableObject {
     ///  - participant: the participant whose track visibility would be changed.
     ///  - isVisible: whether the track should be visible.
     public func changeTrackVisibility(for participant: CallParticipant, isVisible: Bool) {
-        if !isVisible {
```

Comment:
While some of this stuff might not be needed anymore, with the introduction of the visibility modifier, we need to test all cases. This code was here because of tracks being hidden when changing layouts. Did you test that everything works as expected?

Reply:
Tests so far haven't yield anything concerning but i'll run another round with this one and the comment above just to be sure.

Reply:
I had one case where one track never got enabled. Hard to tell if it was us or the SFU.

Reply:
That was related to this #191